mysqltest/mysqltest.go

```diff
@@ -31,13 +32,21 @@ type ConnInfo struct {
 // MustStart starts a MySQL server, or panics if there was an error.
 func MustStart(opts ...Option) (ConnInfo, io.Closer) {
 	conf := buildConfig(opts...)
-	ci, closer, err := start(conf)
-	if err != nil {
-		// The Closer must be called even if there's an error, to clean up the docker container that may
-		// exist.
-		_ = closer.Close()
-		panic(err)
+	driver := (&containertest.MySQL{}).WithVersion(conf.mySQLVersion)
+	realOpts := make([]containertest.Option, 0, 2)
```

Comment:
`realOpts` is perhaps a bit confusing as a name, because it's not immediately clear why they're more real than the other options. Maybe `ctOpts` or `translatedOpts` or something?

Reply:
Changed